libs/components/indicators/src/lib/modules/expansion-indicator/expansion-indicator.modern.component.scss

```diff
@@ -11,6 +11,10 @@
   --sky-override-expansion-indicator-size-half: 13px;
 }
 
+@include compatMixins.sky-modern-v2(':host') {
+  line-height: 1;
```

Comment:
This was causing the expansion indicator to be larger than its inner "icon". I went ahead and set the line-height to 1 to fix that in v2 - but I didn't modify what happens in default and current modern.

Reply:
Done